src/Map/src/Twig/UXMapComponentListener.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+use Symfony\UX\TwigComponent\Event\PreCreateForRenderEvent;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class UXMapComponentListener
+{
+    public function __construct(
+        private MapRuntime $mapRuntime,
+    ) {
+    }
+
+    public function onPreCreateForRender(PreCreateForRenderEvent $event): void
+    {
+        if ('ux:map' !== strtolower($event->getName())) {
+            return;
+        }
+
+        $attributes = $event->getInputProps();
+        $map = array_intersect_key($attributes, array_flip(['zoom', 'center', 'markers']));
```

Comment:
Can't we directly replace `array_flip(['zoom', 'center', 'markers'])` by `['zoom' => 0, 'center' => 1, 'markers' => 2])`? Since the output will always be identical

Reply:
I'm curious, why so ?

Reply:
We can remove an useless function call, but I don't know if PHP/OPCache already optimize this call since we pass a static array.

Reply:
Ok i'll update it for the 3ops win 

(even if i'm really not a big fan if nano-optimizations like this :) )